helix-view/src/ui/markdown.rs

```diff
@@ -264,14 +264,14 @@ impl Markdown {
                             CodeBlockKind::Fenced(language) => language,
                             CodeBlockKind::Indented => "",
                         };
-                        let tui_text = highlighted_code_block(
+                        let helix_tui_text = highlighted_code_block(
```

Comment:
this and the one below don't need to be changed
```suggestion
                        let tui_text = highlighted_code_block(
```